test/command/test-base.rb

```diff
@@ -122,6 +122,132 @@ def test_pretty_print
+{"_key":"http://example.org/","title":"This is test record 1!"}
+        OUTPUT
+
+        assert_equal(expected.chomp, load.to_elasticsearch_format)
+      end
+
+      def test_curly_brackets_format
+        load = Groonga::Command::Base.new("load",
+                                          :table => "Site",
+                                          :values => <<-VALUES)
+  [
+  {"_key": "http://example.org/", "title": "This is test record 1!"}
+  ]
+        VALUES
+
+        expected = <<-OUTPUT
+{"index":{"_index":"site","_type":"groonga"}}
+{"_key":"http://example.org/","title":"This is test record 1!"}
+        OUTPUT
+
+        assert_equal(expected.chomp, load.to_elasticsearch_format)
+      end
+    end
+
+    sub_test_case("multiple_records") do
+      def test_brackets_format
+        load = Groonga::Command::Base.new("load",
+                                          :table => "Site",
+                                          :values => <<-VALUES)
+  [
+  ["_key","title"],
+  ["http://example.org/","This is test record 1!"],
+  ["http://example.net/","This is test record 2!"]
+  ]
+        VALUES
+
+        expected = <<-OUTPUT
+{"index":{"_index":"site","_type":"groonga"}}
+{"_key":"http://example.org/","title":"This is test record 1!"}
+{"_key":"http://example.net/","title":"This is test record 2!"}
+        OUTPUT
+
+        assert_equal(expected.chomp, load.to_elasticsearch_format)
+      end
+
+      def test_curly_brackets_format
+        load = Groonga::Command::Base.new("load",
+                                          :table => "Site",
+                                          :values => <<-VALUES)
+  [
+  {"_key": "http://example.org/", "title": "This is test record 1!"},
+  {"_key": "http://example.net/", "title": "This is test record 2!"}
+  ]
+        VALUES
+
+        expected = <<-OUTPUT
+{"index":{"_index":"site","_type":"groonga"}}
+{"_key":"http://example.org/","title":"This is test record 1!"}
+{"_key":"http://example.net/","title":"This is test record 2!"}
+        OUTPUT
+
+        assert_equal(expected.chomp, load.to_elasticsearch_format)
+      end
+    end
+
+    sub_test_case("options") do
+      expections = {}
+      expections[:version5] = <<-OUTPUT
+{"index":{"_index":"site","_type":"groonga"}}
+{"_key":"http://example.org/","title":"This is test record 1!"}
+      OUTPUT
+      expections[:version6] = expections[:version5]
+      expections[:version7] = <<-OUTPUT
+{"index":{"_index":"site","_type":"_doc"}}
+{"_key":"http://example.org/","title":"This is test record 1!"}
+      OUTPUT
+      expections[:version8] = <<-OUTPUT
+{"index":{"_index":"site"}}
+{"_key":"http://example.org/","title":"This is test record 1!"}
+      OUTPUT
+
+      load = Groonga::Command::Base.new("load",
+                                        :table => "Site",
+                                        :values => <<-VALUES)
+[
+["_key","title"],
+["http://example.org/","This is test record 1!"]
+]
+      VALUES
+
```

Comment:
It's not a good manner to run codes in this context.
Should we use data driver test here? Can we use normal test style here?

```ruby
def setup
  @load = Groonga::Command::Base.new("load",
                                     :table => "Site",
                                     :values => <<-VALUES)
[
["_key","title"],
["http://example.org/","This is test record 1!"]
]
  VALUES
end

sub_test_case(":version") do
  def test_5
    assert_equal(<<-REQUESTS.chomp, @load.to_elasticsearch_format(5))
{"index":{"_index":"site","_type":"groonga"}}
{"_key":"http://example.org/","title":"This is test record 1!"}
    REQUESTS
  end

  def test_6
    assert_equal(<<-REQUESTS.chomp, @load.to_elasticsearch_format(6))
{"index":{"_index":"site","_type":"groonga"}}
{"_key":"http://example.org/","title":"This is test record 1!"}
    REQUESTS
  end

  ...
end
```